src/System.Net.HttpListener/tests/AuthenticationTests.cs

```diff
@@ -47,6 +47,23 @@ public AuthenticationTests()
             }
         }
 
+        [ConditionalTheory(nameof(Helpers) + "." + nameof(Helpers.IsWindowsImplementation))] // Managed implementation connects successfully.
```

Comment:
I'm not sure I follow that: when I run on Windows with no parameters it seems to run against the HTTP.SYS component, not the managed one.
Also, my question was more about the comment "Managed implementation connects successfully." That sounds to me like a bug in the managed implementation.